sys/net/gnrc/network_layer/ipv6/nib/_nib-router.c

```diff
@@ -244,7 +244,8 @@ static gnrc_pktsnip_t *_build_ext_opts(gnrc_netif_t *netif,
         ext_opts = rdnsso;
     }
 #endif  /* CONFIG_GNRC_IPV6_NIB_DNS */
-    if (gnrc_netif_is_6ln(netif)) {
+    if (gnrc_netif_is_6lr(netif)) {
```

Comment:
This now lets the non-6LR-but-6LN-case drop into what was previously a non 6LN-path. Better keep the that case a no-op, IMHO. Just to keep the outer logic as is, i.e., adding an `if (!gnrc_netif_is_6ln(netif))` after the else here.
https://github.com/RIOT-OS/RIOT/blob/15a84eb507994fa74db8f00669459f569624ca61/sys/net/gnrc/network_layer/ipv6/nib/_nib-router.c#L297-L306

Reply:
Done